lang/es.js

```diff
@@ -1,25 +1,25 @@
 /* eslint quotes: 0 */
 
 export default {
-	"components.alert.close": "Alerta de cierre",
+	"components.alert.close": "Cerrar alerta",
 	"components.calendar.notSelected": "No seleccionado.",
 	"components.calendar.selected": "Seleccionado.",
 	"components.calendar.show": "Mostrar {month}",
 	"components.dialog.close": "Cerrar este cuadro de diálogo",
 	"components.input-date.clear": "Borrar",
-	"components.input-date.openInstructions": "",
-	"components.input-date.setToToday": "",
+	"components.input-date.openInstructions": "Utilice el formato de fecha {format}. Presione la flecha hacia abajo o Intro para acceder al minicalendario.",
+	"components.input-date.setToToday": "Establecer para hoy",
 	"components.input-date-time.date": "Fecha",
 	"components.input-date-time.time": "Hora",
 	"components.input-search.clear": "Borrar búsqueda",
 	"components.input-search.search": "Buscar",
-	"components.menu-item-return.return": "",
-	"components.menu-item-return.returnCurrentlyShowing": "{menuTitle}",
+	"components.menu-item-return.return": "Regresa al menú anterior.",
+	"components.menu-item-return.returnCurrentlyShowing": "Regresa al menú anterior. Está viendo {menuTitle}.",
 	"components.meter-mixin.commaSeperatedAria": "{term1}, {term2}",
 	"components.meter-mixin.fraction": "{x}∕{y}",
 	"components.meter-mixin.progressIndicator": "Indicador de progreso",
 	"components.more-less.less": "menos",
 	"components.more-less.more": "más",
-	"components.tabs.next": "Desplácese a la derecha",
```

Comment:
"Scroll right" -> "Scroll forward" and "Scroll left" -> "Scroll back". We must have changed the English translation here?

Reply:
ah interesting - yeah we must have